frontend/src/views/SecretMainPage/components/CreateSecretForm/CreateSecretForm.tsx

```diff
@@ -73,6 +75,20 @@ export const CreateSecretForm = ({
     }
   };
 
+  const handlePaste = (e: ClipboardEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    const pastedContent = e.clipboardData.getData('text');
```

Comment:
Please use double quotes ("), instead of single quotes ('). 

Single quotes will result in a linting error.